src/lib/components/footer/QFooter.svelte

```diff
@@ -1,36 +1,76 @@
 <script lang="ts">
-  import { useSize } from "$lib/composables";
+  import { getContext, onDestroy, onMount, untrack } from "svelte";
   import QContext from "$lib/classes/QContext.svelte";
-  import type { LayoutContext } from "../layout/QLayout.svelte";
+  import QScrollObserver from "$lib/classes/QScrollObserver.svelte";
+  import type { QLayoutProps } from "$components/layout/props";
+  import QToolbar from "$components/toolbar/QToolbar.svelte";
+  import type { AppbarContext } from "../layout/QLayout.svelte";
   import type { QFooterProps } from "./props";
 
+  const ID = Date.now();
```

Comment:
Because it's an absolute constant, like you would have environment variables be uppercase too. I can change it to be lowercase though if you want

Reply:
Mhh, that's debatable. Imo it should be `const id = Date.now()`, because it's not a _hard coded_ constant (which would be in `SCREAMING_SNAKE_CASE` typically). Also, the name `id` might not be ideal, unless we're talking about an element id, but that's up to you.